Formula/ksh.rb

```diff
@@ -1,42 +1,29 @@
 class Ksh < Formula
-  desc "KornShell, ksh93"
+  desc "KornShell, ksh93u+m"
```

Comment:
Does the version here matter to users?

Reply:
Probably not.  The ksh93 fans might see ksh93u+m and get excited, but beyond that, no. :)